sqs/src/main/scala/akka/stream/alpakka/sqs/impl/SqsSourceStage.scala

```diff
@@ -116,3 +119,11 @@ import scala.collection.JavaConverters._
       )
     }
 }
+
+object SqsSourceStage {
+
+  private implicit class ReceiveMessageExtension(request: ReceiveMessageRequest) {
```

Comment:
I'd like to be it explicit when constructing the `ReceiveMessageRequest` instead.